SPECS/rubygem-json/rubygem-json.spec

```diff
@@ -14,6 +14,8 @@ Source0:        https://github.com/flori/json/archive/refs/tags/v%{version}.tar.
 BuildRequires:  git
 BuildRequires:  ruby
 Requires:       ruby(release)
+Provides:       json = %{version}-%{release}
```

Comment:
Could we remove these plain `Provides: json` and the like (`bigdecimal`, `io-console`, etc.)? Or replace them with the names we used to have in `ruby.spec`, so `rubygem-bigdecimal`, `rubygem-io-console`, `rubygem-[other]`? The current names seem too generic and I'm worried they might cause issues at some point. I haven't gone through all specs, but it looks like different `(Build)Requires` go for `rubygem([name])` anyway, so we should be fine.

Reply:
Yep, I'll replace them with what we had in ruby.spec before. Thanks